package.json

```diff
@@ -152,6 +152,7 @@
     "purify-css": "1.2.5",
     "raml-validator-loader": "0.1.11",
     "react-addons-test-utils": "15.6.2",
+    "react-popper": "2.2.3",
```

Comment:
Why do we need new dependency?

Reply:
it looks like we need to declare this explicitly here as else npm resolves that to an older version that does not use hooks yet and breaks.

Reply:
we just learned that we had a dependency that relied on popper.js 1.x, which is why i initially had to add this explicitly.

now that dependency is gone. 🎉